src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -179,6 +179,49 @@
 	color: var(--vscode-textPreformat-foreground);
 }
 
+.interactive-item-compact {
```

Comment:
Write these like `.interactive-item-container.interactive-item-compact ...`, that will make it more clear that it's a "this or that" style and won't break if we rearrange the css file and the order of these rules changes

Reply:
Added!